common/defs-flags.go

```diff
@@ -0,0 +1,60 @@
+package common
+const (
+	// Shared flag definitions
+	FlagEnvDefinition = "Name of the environment to read environmental variables from."
```

Comment:
why separating the definitions? These don't seem to be re-used anywhere

Reply:
Some are repeated. This was mostly to make it easier to find what needs to be changed (and can be useful should the CLI ever support different written languages).

Reply:
Last job I worked in had a heavy focus on optimizing for the translators, so force of habit.

Reply:
right makes sense